spec/models/tag_spec.rb

```diff
@@ -168,6 +168,39 @@ def expect_tag_update_flag_in_redis_to_be(flag)
     expect(tag.errors[:name].join).to match(/too long/)
   end
 
+  context "tags with blank names are invalid" do
+    [
+      ["200B".hex].pack("U"), # zero width space
+      ["200D".hex].pack("U"), # zero width joiner
+      ["2060".hex].pack("U"), # word joiner
+      (["200B".hex].pack("U") * 3),
+      ["200D".hex, "200D".hex, "2060".hex, "200B".hex].pack("U")
+    ].each do |tag_name|
+      tag = Tag.new
+      tag.name = tag_name
```

Comment:
Could the following work instead?
```suggestion
      let(:tag) { build(:tag, name: tag_name) }
```

Reply:
The emoji test starts failing with this and I can't tell why. The `expect(tag.name).to eq(tag_name)` line fails for all but the _last_ tag (it always returns the scientist for the tag.name), even if I attempt something like destroying the tag in the it block.

Reply:
It always fails on the first two tags for the emojis. I guess something is going wrong with the `let` memoization. So I would rather not change this at all.

Reply:
Fair enough!